### PR TITLE
Feature/ssh server config

### PR DESCRIPTION
**Pull request by empwilli**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [kirkstone-0.19.13] Q3 2023
```
Comment (mlilien) [inline CHANGELOG.md:8]: with this feature more a 0.20.0

File: recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd [hunk 2]
```diff
@@ -1,2 +1,3 @@
 d /home/omnect/.ssh 0700 omnect omnect -
 f /home/omnect/.ssh/authorized_keys 0600 omnect omnect -
+d /run/omnect-device-service/ssh-tunnel 0747 omnect_device_service omnect_device_service 1h -
```
[the remark this answers is not in the document]
Comment (empwilli) [reply recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd:3]: I'm also not super happy with this solution. We have to handle/store three files here: private and public key (currently generated as ssh_tunnel_user), and the device certificate (currently stored by omnect-device-user as this greatly shortens the code/simplifies the logic). In the end the ssh_tunnel_user needs access to a) the cert, b) the priv key, i.e. with basic unix access permissions they must either be (at least) x04 or be owned by the ssh_tunnel_user. Chowning would (IMHO) lead to quite ugly code. 

Any suggestions on how to properly (and ideally nicely) assigning as little permissions as possible?

[the remark this answers is not in the document]
Comment (mlilien) [reply recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd:3]: oh, okay ... I'm just in the process to change the creation of /run/omnect-device-service at every boot via initramfs.
the dir would belong to omnect-device-service:omnect-device-service with 0775 permissions.
so you should be able to add your user into group omnect-device-service and then create a subdir with no access for others.

(https://github.com/omnect/meta-omnect/pull/326)

[the remark this answers is not in the document]
Comment (mlilien) [reply recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd:3]: proposal: i approve it and will change it in #326

**Review by mlilien — APPROVED** [review]

**Pull request merged**